OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs

```diff
@@ -47,8 +47,8 @@ public class ProductionQueueInfo : ITraitInfo
 		[Desc("Maximum number of items that can be queued across all actor types (0 = infinite).")]
 		public readonly int QueueLimit = 0;
 
-		[Desc("The build time is multiplied with this value on low power.")]
-		public readonly int LowPowerSlowdown = 3;
+		[Desc("The build time is multiplied with this percentage on low power.")]
+		public readonly int LowPowerModifier = 100;
```

Comment:
Can you please do this across two commits? The first to refactor `Slowdown` to `Modifier` (keeping the default at 300), and then a second to change the default?  Keeping refactoring and behaviour changes in separate commits makes it a lot easier to understand commits when looking back from the far future.